### ISSUE TITLE
Importing of logs issue

### ISSUE THREAD

**Issue by bobl61** [A1mDev/hlstatsx-community-edition#8]
After resetting the stats and importing old logs, the stats are all messed up. Numerous players have no "last connect" stat. If you change the sort order (IE: accuracy, kills, etc.) the ranks change. All players show 98% activity (hlstats-awards.pl run daily).

![2023-07-11_073443](https://github.com/A1mDev/hlstatsx-community-edition/assets/107001969/0bdfef38-ddb4-4d41-b09b-a397e06c62c1)

![2023-07-11_073556](https://github.com/A1mDev/hlstatsx-community-edition/assets/107001969/5f68b49b-880d-4f88-8773-0de44fce6f44)

![2023-07-11_073738](https://github.com/A1mDev/hlstatsx-community-edition/assets/107001969/fbc4c7f5-6ab6-409a-8647-43b2884ebdc2)

**Comment by A1mDev** [comment]
did you import the database again after the reset or what?

**Comment by bobl61** [comment]
No, I did not import the old database. I reset the stats (full reset via admin panel), then imported the old logs.
